### ISSUE TITLE
Add header and scope to file upload tables

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#175]
SODA recommends changing the first `<td>` cell in each row to `<th scope="row">`.

See the two-header table examples: https://dequeuniversity.com/class/semantic-structure/tables/simple and https://www.w3.org/WAI/tutorials/tables/two-headers/.

Additionally, the above examples indicate that `scope="col"` should be added to each `<th>` in the `<thead>` header row (i.e., to the Supplemental File, Type, Size, Upload Date, and Remove `<th>` elements). 

Note: I tested the above approach (including changing the "File description" row to `<th scope="row">`) and it seemed to work as expected.

### Reported by SODA
> [Summary]
> Data table has missing or incomplete header cell markup.
> 
> [Description]
> The data table has missing or incomplete header cell markup.
> The data table for multiple doc uploads and the dissertation upload are missing a row header. The name of the document (first column) should be the row header in both cases. 
> 
> [Impact]
> Critical
> 
> [Recommended to fix]
> Change the first cell in each row to be a row header: 
> 
> `<th scope="row"><a href="../Page1.pdf">Page1.pdf</a></th>`
> 
> Rule : 
> Table headers MUST be designated with `<th>`.
> 
> How To Fix: 
> Fix this issue by doing ALL of the following:
> 1. Mark all column header and/or row header cells with a `<th>` element.
> 2. Ensure that header cell and data cell relationships are correctly conveyed according to the complexity of the table.
> 
> Reference: 
> Deque University: https://dequeuniversity.com/class/semantic-structure/tables/
> W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/tables/
> 
> Background: 
> People who are blind cannot see the organizational structure of a table with data arranged in rows and columns with corresponding header cells. In order for screen reader users to understand the logical relationships of data arranged in a table, tables need HTML markup that indicates header cells and data cells and defines their relationship. When tables are marked correctly, screen reader users are able to navigate data tables from cell to cell, in a multi-directional way (up, down, right, left), much like navigating a spreadsheet. As they move from cell to cell, screen readers will read the associated header labels.
>
